src/constructs/aws/StaticWebsite.ts

```diff
@@ -17,7 +19,18 @@ export class StaticWebsite extends StaticWebsiteAbstract {
         protected readonly configuration: CommonStaticWebsiteConfiguration,
         protected readonly provider: AwsProvider
     ) {
-        super(scope, id, configuration, provider);
+        // we need to build specific bucket props to pass to the abstract static website constructor
+        const bucketProps: BucketProps = {
+            // Enable static website hosting
+            websiteIndexDocument: "index.html",
+            websiteErrorDocument: configuration.errorPage,
```

Comment:
isn't this duplicate with https://github.com/getlift/lift/blob/f715c8691a622092b50760c37c53344b3aa325b8/src/constructs/aws/abstracts/StaticWebsiteAbstract.ts#L269 ?